src/tls/README.md

```diff
@@ -0,0 +1,65 @@
+cockpit TLS proxy
+=================
+
+`cockpit-tls` is a TLS terminating HTTP proxy for cockpit-ws. TLS termination
+should not be done directly in `cockpit-ws`, as it is hard to audit and trust:
```

Comment:
It is not super clear what the "it" in "it is hard to audit and trust" refers to.  I first read it as referring to TLS termination, but it really< refers to cockpit-ws, right?

Reply:
Correct. would "in cockpit-ws, as that is hard to audit and trust" be better?

Reply:
I changed it to "that", but I leave it for you to resolve or follow up.